### PR TITLE
Update cli.adoc

### PR DESCRIPTION
**Pull request by Chr1st1anSears**
Starting my getting started w CLI doc draft with this first commit

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started/cli.adoc [hunk 1]
```diff
@@ -1 +1,78 @@
 = Getting started with the CLI
+
+While {ProductName} offers a powerful UI, users may also want to access the platform through their command line interface (CLI). For users to gain this access, a Platform Engineer (PE) must first configure their instance of {ProductName} to support CLI access. They must also draft a template `kubeconfig` file for their organization. Afterwards, users can edit that template to create a `kubeconfig` file specific to themselves. Then, they use that file to gain access to {ProductName} through the CLI.
```
Comment (ralphbean) [inline docs/modules/ROOT/pages/getting-started/cli.adoc:3]:
```suggestion
While {ProductName} offers a powerful UI, users may also want to access the platform through the `kubectl` command line interface (CLI). For users to gain this access, a Platform Engineer (PE) must first configure their instance of {ProductName} to support CLI access. They must also draft a template `kubeconfig` file for their organization. Afterwards, users can edit that template to create a `kubeconfig` file specific to themselves. Then, they use that file to gain access to {ProductName} through the CLI. 
```

**Review by ralphbean — APPROVED** [review]

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Review by ernesgonzalez33 — COMMENTED** [review]
I'm missing the instructions to access shared workspaces. Are those going to be somewhere else?

**Comment by Chr1st1anSears** [comment]
> I'm missing the instructions to access shared workspaces. Are those going to be somewhere else?

I believe so. We have a number of other docs coming down the pipeline.

**Pull request merged**